src/main/java/club/sk1er/hytilities/Hytilities.java

```diff
@@ -121,4 +260,50 @@ public boolean isLoadedCall() {
     public void setLoadedCall(boolean loadedCall) {
         this.loadedCall = loadedCall;
     }
+
+    public boolean isValidConfigVersion() {
+        return validConfigVersion;
+    }
+
+    public Map<String, String> getRestyleMeta() {
+        return restyleMeta;
+    }
+
+    public static String fetchString(String url) throws IOException {
```

Comment:
update to autogg's version where it'll reuse the connection
put method above getters